### DISCUSSION TITLE
Is this software Owned, Accessed, Developed, etc... by the CCP?

### DISCUSSION THREAD

**Discussion by Neipas09** [rustdesk/rustdesk#1159] [Q&A]
I'm seeing online that RustDesk is chinese developed software.
Last i knew, everything developed/created in China is owned by the CCP.
Does that include this software?

Pretty scary thought that the CCP could have access to your computer without you knowing it, or monitoring your connections that you make through the "You can use our public rendezvous/relay server"

**Comment by rustdesk** [comment]
We are a team from Singapore. Sorry, I will lock your discussion soon, no politics allowed here.
